doc/rtd/howto/run_cloud_init_locally.rst

```diff
@@ -78,8 +84,9 @@ Boot the cloud image with our configuration, :file:`seed.img`, to QEMU:
 .. code-block:: shell-session
 
     $ qemu-system-x86_64 -m 1024 -net nic -net user \
-        -hda jammy-server-cloudimg-amd64.img \
-        -hdb seed.img
+        -drive file=jammy-server-cloudimg-amd64.img,index=0,format=qcow2,media=disk \
+        -drive file=seed.img,index=1,media=cdrom \
+        -machine accel=kvm:tcg
```

Comment:
is there any benefit to explaining what each of the various args passed are?

i feel like that might be out of scope for this since this is just cloud-init specific docs.

Reply:
such as:

---

### Explanation

#### `qemu-system-x86_64`
- Emulates a 64-bit x86 machine.

#### `-m 1024`
- Allocates 1024 MB of RAM to the VM.

#### `-net nic`
- Adds a network interface card to the VM.

#### `-net user`
- Uses QEMU's user mode networking for basic network connectivity.

#### `-hda jammy-server-cloudimg-amd64.img`
- Sets the primary hard disk using the specified image.

#### `-hdb seed.img`
- Sets the secondary hard disk using the specified image, often for cloud-init data.

#### `-drive file=jammy-server-cloudimg-amd64.img,index=0,format=qcow2,media=disk`
- Specifies the primary drive in detail:
  - `file`: Image file to use.
  - `index`: Drive index (0 for primary).
  - `format`: Image format (QCOW2).
  - `media`: Type of drive (disk).

#### `-drive file=seed.img,index=1,media=cdrom`
- Specifies the secondary drive in detail:
  - `file`: Image file to use.
  - `index`: Drive index (1 for secondary).
  - `media`: Type of drive (CD-ROM).

#### `-machine accel=kvm:tcg`
- Uses KVM for hardware acceleration if available, otherwise TCG for software emulation.

Reply:
Yes, I think this is probably out of scope for this tutorial to tell folks how to use QEMU command line and it probably should be left as an exercise to the user to RTFM to better understand the implications of qemu commandline in this example.